spec/alphabetic_bullets_spec.rb

```diff
@@ -132,6 +132,33 @@
       TEXT
     end
 
+    it 'does not add a new bullet when mixed case' do
+      test_bullet_inserted('not a bullet', <<-INIT, <<-EXPECTED)
+        # Hello there
+        Ab. this is the first bullet
+      INIT
+        # Hello there
+        Ab. this is the first bullet
+        not a bullet
+      EXPECTED
+    end
+
+    # it 'does not add a new alpha bullet with wrapped lines' do
+    # # TODO: maybe take guidance from Pandoc and require two spaces after the
+    # closure to allow us to differentiate between bullets and abbreviations
+    # and words. Might also consider only allowing single letters.
+    #   test_bullet_inserted('not a bullet', <<-INIT, <<-EXPECTED)
+    #     # Hello there
+    #     a. first bullet might not catch
+    #     	me. second line.
+    #   INIT
+    #     # Hello there
+    #     a. first bullet might not catch
+    #     	me. second line.
+    #     not a bullet
+    #   EXPECTED
+    # end
+
```

Comment:
That's a bummer, I think I have a solution though, for alphabetic bullets, if nested we can check that the first one is `a.` or `A.` - not necessary to fix this in this PR but good to keep this here as a TODO.

Reply:
I had a similar thought about checking to see if an item is the first in the list, and if not, check to see if the previous sibling bullet is one increment lower. This solution breaks if the bullets are not sequential because of reordering, inserting new bullets above the current one, etc., but might be better than nothing.

Another option would be to follow the Pandoc-Markdown rules for [ordered lists](https://pandoc.org/MANUAL.html#ordered-lists), which only allow single letter bullets (except roman numerals) and require two spaces after capital letters to disambiguate from initialized names (e.g., `G. Washington` vs. `G.  Washington`).